src/main/scala/viper/gobra/Gobra.scala

```diff
@@ -192,8 +192,9 @@ class Gobra extends GoVerifier with GoIdeVerifier {
       } else {
         // our current "merge" strategy for potentially different, duplicate, or even contradicting configurations is to concatenate them:
         val args = configs.flatMap(configString => configString.split(" ")).toList
+        val actualArgs = args filter (_ != Config.headerTag)
```

Comment:
I'm not super excited about this solution as I'm a bit afraid this will end up being a long list of some special cases in the future.

What about [something more Go like](https://www.digitalocean.com/community/tutorials/building-go-applications-for-different-operating-systems-and-architectures), e.g. `\\ +verify gobra`, and keep it completely separate from our infile configs?